cmd/erasure-server-pool.go

```diff
@@ -2164,31 +2168,42 @@ func (z *erasureServerPools) Health(ctx context.Context, opts HealthOptions) Hea
 		usingDefaults = true
 	}
 
+	var maximumWriteQuorum int
+	for _, writeQuorum := range poolWriteQuorums {
+		if maximumWriteQuorum == 0 {
```

Comment:
Should these conditions be merged in one using `||` ?

Reply:
how? @shtripat

Reply:
```
                if maximumWriteQuorum == 0 || writeQuorum > maximumWriteQuorum {
	                maximumWriteQuorum = writeQuorum
		}
```

Reply:
Oh no it is on purpose written the way it is for readability.

Reply:
Also it's existing code.

Reply:
Ack